### ISSUE TITLE
Flaky tests (track flaky tests for further investigation)

### ISSUE THREAD

**Issue by Moumouls** [parse-community/parse-server#7094]
Postgres: 
```
Failures:
1) Cloud Code cloud jobs should run with master key basic auth
  Message:
    Unhandled promise rejection: error: relation "_SCHEMA" does not exist (line 1180)
```
Link: https://github.com/parse-community/parse-server/runs/1601708329?check_suite_focus=true

```
Failures:
1) Cloud Code cloud jobs should run with master key
  Message:
    Unhandled promise rejection: error: relation "_JobStatus" does not exist (line 1180)
  Stack:
    error properties: Object({ length: 108, severity: 'ERROR', code: '42P01', detail: undefined, hint: undefined, position: '8', internalPosition: undefined, internalQuery: undefined, where: undefined, schema: undefined, table: undefined, dataType: undefined, constraint: undefined, file: 'parse_relation.c', routine: 'parserOpenTable' })
```
https://github.com/parse-community/parse-server/pull/7091/checks?check_run_id=1604729773

Redis:
```
Failures:
1) Cloud Code cloud jobs should run with master key
  Message:
    Unhandled promise rejection: ParseError: 101 Object not found.
  Stack:
    error properties: Object({ code: 101 })
    Error: Object not found.
        at then.then.result (/home/runner/work/parse-server/parse-server/lib/Controllers/DatabaseController.js:49:3084)
        at process._tickCallback (internal/process/next_tick.js:68:7)
```
https://github.com/parse-community/parse-server/pull/7091/checks?check_run_id=1604696128

**Comment by mtrezza** [comment]
Thanks for reporting.

Would you please rephrase this using the issue template?

**Comment by mtrezza** [comment]
Closed in favor of #7180
